tiup/tiup-reference.md

```diff
@@ -20,7 +20,7 @@ You can use the `--help` command to get the information of a specific command. T
 
 ## Options
 
-### -B, --binary
+### --binary
```

Comment:
Do we need to remove `-B/-` in Lines 27 and 28 as well?

Reply:
Done https://github.com/pingcap/docs/pull/11362/commits/0ba3745dd419b660a415c3670cf742d102d55028